pages/changelog.mdx

```diff
@@ -12,6 +12,15 @@ Keep up to date on new product features and enhancements, API docs, tools change
 
 ## Upcoming
 
+### 2023-08-31
+
+**DEPRECATED**
```

Comment:
**SUNSET**

- Dwolla will be retiring the `dwolla.js` library, which supports adding an unverified bank funding source. Check out the official [announcement](https://discuss.dwolla.com/t/may-2023-updates-retiring-dwolla-s-client-side-javascript-library-dwolla-js/8969).
- The legacy dwolla.js will be replaced with functionality that has been added to our [Drop-in Components library](/concepts/drop-in-components#latest-v220). This library will serve as our primary web UI components library and offer an enhanced developer experience. Please refer to our [migration guide](https://discuss.dwolla.com/t/migration-guide-dwolla-js-to-dwolla-web-js/8968) for more details, including alternative solutions.
-  The following API endpoints will be removed as part of the sunset: `/customers/{id}/iav-token` and `/customers/{id}/funding-sources-token`

Reply:
[Applied suggestions from code review](https://github.com/Dwolla/dev-portal/pull/507/commits/4854c1cc3319348121c30188fb039396eda16222)